src/RequestHeaders.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Yiisoft\RequestProvider;
+
+final class RequestHeaders {
+    /**
+     * @param RequestProviderInterface $requestProvider
+     */
+    public function __construct(
+        private readonly RequestProviderInterface $requestProvider
+    ) {
+    }
+
+    /**
+     * @param string $name
+     * @return string|null
+     */
+    public function getHeaderLine(string $name): string|null {
```

Comment:
What about using the default value?
```suggestion
    public function getHeaderLine(string $name, string $default = null): string|null {
```